lib/resque/version.rb

```diff
@@ -1,3 +1,3 @@
 module Resque
-  Version = VERSION = '1.27.4.skroutz.0'
+  Version = VERSION = '1.27.4.skroutz.1.pre'
```

Comment:
Why the `pre`?

Reply:
This was an internal pre-release. Now it is `1.27.4.skroutz.1`